### ISSUE TITLE
[1.7.10]: Null pointer exception when teleporting to Eternal Frost , /cofh tpx 4

### ISSUE THREAD

**Issue by quentin452** [quentin452/1.7.10-Forge-renewed-v1#935]
### Modpack

Biggess Pack Cat Edition V0.98.8DEV

### Game log

```
java.lang.NullPointerException
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: Index: 15 Listeners:
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 0: HIGHEST
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 1: ASM: vazkii.botania.client.core.handler.TooltipHandler@2dca3ab1 onTooltipEvent(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 2: ASM: witchinggadgets.client.ClientEventHandler@6e0133a4 getTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 3: NORMAL
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 4: ASM: lotr.client.LOTRTickHandlerClient@577b16ce getItemTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 5: ASM: minetweaker.mc1710.ForgeEventHandler@4294c8b4 onItemTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 6: ASM: mods.battlegear2.client.BattlegearClientEvents@13d6079a onItemTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 7: ASM: buildcraft.core.list.ListTooltipHandler@29e21c15 itemTooltipEvent(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 8: ASM: com.rwtema.extrautils.modintegration.TConEvents@56b1af90 unstableTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 9: ASM: thaumcraft.common.lib.events.EventHandlerRunic@4ed65f76 tooltipEvent(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 10: ASM: com.thecrafter4000.lotrtc.TinkersMEEvents@5d2901cc addTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 11: ASM: net.bdew.lib.tooltip.TooltipHandler$@4d97f74f handleTooltipEvent(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 12: ASM: forestry.apiculture.EventHandlerApiculture@5a74a3e1 addFrameTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 13: ASM: crazypants.enderio.machine.painter.BlockPaintedGlowstone$PainterTemplate@59bb01c9 onTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 14: ASM: com.brandon3055.draconicevolution.common.handler.MinecraftForgeEventHandler@65ffa5f7 itemTooltipEvent(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 15: ASM: DummyCore.Utils.DummyEventHandler@3af66d12 onDescrAdded(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 16: ASM: com.pam.harvestcraft.PamTooltipEventHandler@7994a7cc onItemTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 17: ASM: INSTANCE onItemTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 18: ASM: micdoodle8.mods.galacticraft.core.event.EventHandlerGC@2b08cb1f onItemTooltipEvent(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 19: ASM: latmod.core.mod.client.LCClientEventHandler@7418eab1 onTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 20: ASM: mariculture.core.gui.GuiItemToolTip@58b1031d addToolTip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 21: ASM: mariculture.fishery.BaitListingsHandler@74b22d4a addToolTip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 22: ASM: tonius.neiintegration.mods.mcforge.MCForgeTooltipHandler@3e184673 onItemTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 23: ASM: openblocks.enchantments.flimflams.LoreFlimFlam$DisplayHandler@218c189d onItemTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 24: ASM: pneumaticCraft.client.ClientEventHandler@310d8a38 onItemTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 25: ASM: lumien.randomthings.Handler.RTEventHandler@25396b36 toolTip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 26: ASM: remoteio.client.handler.TooltipEventHandler@661a62db itemTooltipEvent(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 27: ASM: serverutils.handlers.ServerUtilitiesClientEventHandler@6dce89e2 onTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 28: ASM: com.Zoko061602.SuperTic.SuperEventHandler@6bc0208a Tooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 29: ASM: net.darkhax.wawla.addons.generic.AddonGenericTooltips@5690f89d onTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 30: ASM: pavocado.zoocraftdiscoveries.init.ZCEventHandler@49a6448b getTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 31: ASM: com.gtnewhorizon.gtnhlib.util.AnimatedTooltipHandler renderTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 32: ASM: com.rwtema.extrautils.EventHandlerClient@7bbd7b85 generatorTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 33: ASM: fox.spiteful.avaritia.LudicrousEvents@7e780a49 onTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 34: ASM: buildcraft.builders.BuilderTooltipHandler@3dac3442 itemTooltipEvent(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 35: ASM: blusunrize.immersiveengineering.client.ClientEventHandler@50ea6f8c onItemTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 36: ASM: ganymedes01.etfuturum.core.handlers.ClientEventHandler@2ae7c2f toolTipEvent(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 37: ASM: de.ellpeck.actuallyadditions.mod.event.TooltipEvent@6c879435 onTooltipEvent(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 38: ASM: com.InfinityRaider.AgriCraft.handler.ItemToolTipHandler@1636c725 addSeedStatsTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 39: ASM: com.InfinityRaider.AgriCraft.handler.ItemToolTipHandler@1636c725 addClipperTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 40: ASM: com.InfinityRaider.AgriCraft.handler.ItemToolTipHandler@1636c725 addTrowelTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 41: ASM: akuto2.akutoengine.event.ToolTipEvent@55a128fe tTipEvent(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 42: ASM: fox.spiteful.forbidden.FMEventHandler@26e25b8 onTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 43: ASM: redgear.brewcraft.event.TipHandler@46b2e097 onTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 44: ASM: com.enderio.core.client.handlers.EnchantTooltipHandler@f6d15a8 handleTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 45: ASM: com.enderio.core.client.handlers.OreDictTooltipHandler@432c714b onItemTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 46: ASM: INSTANCE addTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 47: ASM: crazypants.enderio.conduit.item.filter.ClearFilterRecipe@1d25aa74 onTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 48: ASM: crazypants.enderio.machine.ClearConfigRecipe@3eac7093 onTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 49: ASM: ipsis.dyetopia.handler.ItemTooltipHandler@689ec8e onItemToolTip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 50: ASM: net.ender.enderaddons.events.ItemHover@7cd5cccb getEvent(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 51: ASM: makeo.gadomancy.common.events.EventHandlerWorld@26341669 on(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 52: ASM: ganymedes01.ganysend.core.handlers.HandlerEvents@f357249 tooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 53: ASM: vapourdrive.hammerz.handlers.ToolTipHandler@2e962347 itemToolTipAddition(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 54: ASM: com.momnop.interdictionpillar.events.InterdictionPillarEventHandler@6b57dd37 setTooltips(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 55: ASM: invtweaks.forge.ClientProxy@3cce2013 onItemTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 56: ASM: modtweaker2.ClientEvents@702e8fc onDrawTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 57: ASM: ca.wescook.nutrition.events.EventTooltip@178f93d4 tooltipEvent(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 58: ASM: taintedmagic.common.handler.TaintedMagicEventHandler@4e0f2e1a tooltipEvent(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 59: ASM: net.blay09.mods.tcinventoryscan.client.ClientProxy@4f5393df onTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 60: ASM: dk.mrspring.kitchen.event.SandwichableTooltipEvent@739e4cef handleToolTip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 61: ASM: squeek.tictooltips.TooltipHandler@5ad057db onItemTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 62: ASM: com.samravenii.ultracompression.handler.UCompEventHandler@15f37897 onItemTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 63: ASM: wildycraft.RSToolTipEvent@42d258a1 toolTipGenerated(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 64: ASM: tconstruct.achievements.AchievementEvents@5f7f2d1b tooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 65: ASM: tconstruct.tools.TFToolEvents@5f6a28ba tooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 66: ASM: eternalfrost.enchant.EnchantedItem@6b3ea5b onTick(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 67: ASM: alkalus.main.core.util.TooltipHandler@a49b350 onItemTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 68: LOWEST
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 69: ASM: net.glease.tc4tweak.ClientProxy@4c2fd94 onTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 70: ASM: net.doubledoordev.d3core.util.ForgeEventHandler@3b531ab2 itemTooltipEventHandler(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 71: ASM: flaxbeard.steamcraft.handler.SteamcraftEventHandler@8158957 plateTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 72: ASM: makeo.gadomancy.common.events.EventHandlerGolem@587ffdd3 on(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 73: ASM: travellersgear.client.ClientProxy@1db3a901 onTooltip(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
[20:23:06] [ForkJoinPool.commonPool-worker-3/ERROR] [FML]: 74: ASM: iguanaman.hungeroverhaul.util.IguanaEventHook@1d11e24f renderTooltips(Lnet/minecraftforge/event/entity/player/ItemTooltipEvent;)V
```

### Description

Null pointer exception when teleporting to Eternal Frost , /cofh tpx 4

edit : maybe this mod is a suspect "squeek.tictooltips.TooltipHandler"

**Comment by quentin452** [comment]
It seem thats fixed in Biggess Pack Cat Edition V0.98.8

**Comment by quentin452** [comment]
fixed in Biggess Pack Cat Edition V0.98.8
